test/ch/aggregation_test.exs

```diff
@@ -0,0 +1,100 @@
+defmodule Ch.AggregationTest do
+  use ExUnit.Case
+
+  setup do
+    conn = start_supervised!(Ch)
+
+    create_table = """
+    create table ch_candle_fragments (
+      ticker LowCardinality(String),
+      time DateTime('UTC') CODEC(Delta, Default),
+      high Float64 CODEC(Delta, Default),
+      open Float64 CODEC(Delta, Default),
+      close Float64 CODEC(Delta, Default),
+      low  Float64 CODEC(Delta, Default),
+    ) ENGINE = MergeTree()
+    ORDER BY (ticker, time)
+    """
+
+    Ch.query!(conn, create_table)
+    on_exit(fn -> Ch.Test.sql_exec("drop table ch_candle_fragments") end)
+
+    create_table = """
+    CREATE MATERIALIZED VIEW ch_candles_one_hour_amt
+    (
+      ticker LowCardinality(String),
+      time DateTime('UTC') CODEC(Delta, Default),
+      high SimpleAggregateFunction(max, Float64) CODEC(Delta, Default),
+      open AggregateFunction(argMin, Float64, DateTime('UTC')),
+      close AggregateFunction(argMax , Float64, DateTime('UTC')),
+      low SimpleAggregateFunction(min, Float64) CODEC(Delta, Default)
+    )
+    ENGINE = AggregatingMergeTree()
+    ORDER BY (ticker, time)
+    AS
+    select
+    t.ticker as ticker,
+    toStartOfHour(t.time) as time,
+    max(t.high) as high,
+    argMinState(t.open, t.time) as open,
+    argMaxState(t.close, t.time) as close,
+    min(t.low) as low
+    from ch_candle_fragments t
+    group by ticker, time
+    """
+
+    Ch.query!(conn, create_table)
+    on_exit(fn -> Ch.Test.sql_exec("drop table ch_candles_one_hour_amt") end)
+
+    insert_query = """
+      insert into ch_candle_fragments
+        (ticker, time, high, open, close, low)
+      VALUES
+      -- 1681410780  UTC
+      -- 1681410840 '2023-04-13 18:34:00' UTC
+      -- 1681410900 '2023-04-13 18:35:00' UTC
+      -- 1681410960 '2023-04-13 18:36:00' UTC
```

Comment:
now that CI passes, can we remove these comments?

Reply:
Done